sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeFormatter.scala

```diff
@@ -49,8 +49,11 @@ private class CodeFormatter {
   private var currentLine = 1
 
   private def addLine(line: String): Unit = {
-    val indentChange =
+    val indentChange = if (line.startsWith("/*") || line.startsWith("//")) {
```

Comment:
btw would be great to add unit tests for this change, once we do it properly

CodeFormatterSuite